doc/src/manual/functions.md

```diff
@@ -140,6 +146,31 @@ Int8
 This function will always return an `Int8` regardless of the types of `x` and `y`.
 See [Type Declarations](@ref) for more on return types.
 
+Incidentally, the type of the arguments can also be specified similarly,
+but this is the topic of the [Methods](@ref) chapter.
+
+### Returning nothing
+
+Some functions are used only for their side effects, and do not need to return a value.
+In these cases, the Julia convention is to return the value [`nothing`](@ref):
+
+```julia
+function printx(x)
+    println("x = $x")
```

Comment:
Might be good to use an expression here that doesn't already evaluate to `nothing`.

Reply:
Ok, I understand you see this example as *contrived* because it would be equivalent without the `return nothing` line, is this correct? Indeed I hadn't thought of that. One way would be to write down this remark. As for another example, do you have any idea?

Instead of a display function (I guess that all display functions other than `println` will also return `nothing`), we could use a function which sets some global variable like
```julia
function setflag(a)
    global flag
    flag = a
    return nothing
end
```
However, this kind of function may more likely return the new value rather than `nothing`. Also, the `global` keyword is introduced two chapters later (if we consider the manual to be a sequential reading). What do you think?